vispy/visuals/visual.py

```diff
@@ -30,6 +30,11 @@ class Visual(object):
     bounds_change : Event
         Emitted when the bounds of the visual have changed.
 
+    Notes
+    -----
+    -   When used in the scenegraph, all Visual classes are mixed with
+    vispy.scene.Node in order to implement the methods, attributes and
```

Comment:
single-backticks around `vispy.scene.Node` here. If you're really curious see:

https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt

But the short version is, it will make it get properly linked in our docs.